scio-core/src/main/scala/com/spotify/scio/io/PubsubIO.scala

```diff
@@ -45,6 +45,9 @@ sealed trait PubsubIO[T] extends ScioIO[T] {
 object PubsubIO {
   final case class ReadParam(isSubscription: Boolean)
 
+  final case class WriteParam(maxBatchSize: Option[Int] = None,
+                              maxBatchBytesSize: Option[Int] = None)
+
   def apply[T: ClassTag: Coder](name: String,
```

Comment:
@jto @ClaireMcGinty it seems that `idAttribute` and `timestampAttribute` are good candidates to be part of `ReadParam` and `WriteParam` types WDYT?

Reply:
yes indeed